backend/yugioh/views.py

```diff
@@ -37,3 +40,32 @@ class YugiohCardInSetViewSet(viewsets.ModelViewSet):
     filterset_class = YugiohCardInSetFilter
     http_method_names = ['get']
 
+
+@extend_schema(tags=['Best Seller Cards'])
+class BestSellerCardListView(generics.ListAPIView):
+    """
+    Viewset for API endpoint that shows 3 bestseller cards for all times.\n
+    The bestseller card is defined as the card that has been ordered the most.\n
+        - The endpoint is /api/bestseller/
+    """
+    serializer_class = BestSellerCardSerializer
+    authentication_classes = []
+    permission_classes = []
+    pagination_class = None
+
+    def get_queryset(self):
+
+        top_cards = OrderItem.objects.values('listing__card').annotate(
+            order_count=Count('quantity')).order_by('-order_count')[:10]
+
+        top_cards_ids = [card['listing__card'] for card in top_cards]
+
+        queryset = Listing.objects.filter(card__in=top_cards_ids
```

Comment:
Are you taking the `completed` orders only? To me, only completed orders are real sales.

Reply:
Now i take only the orders with status ```completed```.